src/ucp/wireup/select.c

```diff
@@ -1063,6 +1056,23 @@ ucs_status_t ucp_wireup_select_lanes(ucp_ep_h ep, const ucp_ep_params_t *params,
                                                           lane_descs,
                                                           key->num_lanes);
 
+    /* add to map first UCP_MAX_OP_MDS fastest MD's */
+    for (i = 0, rma_bw_md_count = 0;
+         key->rma_bw_lanes[i] != UCP_NULL_LANE && rma_bw_md_count < UCP_MAX_OP_MDS; i++) {
```

Comment:
pls add ( ) around both parts of the && expression

Reply:
done